bgpd/bgp_attr.c

```diff
@@ -4682,6 +4682,11 @@ bgp_size_t bgp_packet_attribute(struct bgp *bgp, struct peer *peer,
 		 * there! (JK)
 		 * Folks, talk to me: what is reasonable here!?
 		 */
+
+		/* Make sure dup aspath before the modification */
+		if (aspath == attr->aspath) {
```

Comment:
Also, please strip `{}` for short branches.

Reply:
Updated. Thx.